source/documentation/dns/register_defensive_domain.html.md.erb

```diff
@@ -0,0 +1,79 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: Register New Defensive Domain
+last_reviewed_on: 2024-10-16
+review_in: 6 months
+---
+
+# Register New Defensive Domain
+
+This document explains the process of registering a defensive domain at the Ministry of Justice.
+
+Security owns the policy for [Defensive Domain Registrations](https://security-guidance.service.justice.gov.uk/defensive-domain-registration/#defensive-domain-registrations). 
+
+Operations Engineering doesn't own the decisions around what should be a defensive domains, or what requires a defensive domains. We just ensure that the policy is implemented when a requirement is justified.
```

Comment:
```suggestion
The [Defensive Domain Registrations policy](https://security-guidance.service.justice.gov.uk/defensive-domain-registration/#defensive-domain-registrations) is maintained by the Security team. Operations Engineering is not involved in deciding which domains require defensive registration or determining the criteria for such registrations. Our role is to implement the policy once a justified requirement is presented.
```

Reply:
How do we get in touch with security? Who are security? What the hell is a defensive domain? :)